tmt/utils.py

```diff
@@ -3600,3 +3670,136 @@ def __init__(
 
         kwargs.setdefault('logger', logger)
         super().__init__(node=node, **kwargs)
+
+
+FieldCLIOption = Union[str, Sequence[str]]
+
+
+@overload
+def field(
+        *,
+        default: bool,
+        # Options
+        option: Optional[FieldCLIOption] = None,
+        is_flag: bool = True,
+        choices: Union[None, Sequence[str], Callable[[], Sequence[str]]] = None,
+        multiple: bool = False,
+        metavar: Optional[str] = None,
+        help: Optional[str] = None,
+        # Input data normalization
+        # normalize: Optional[NormalizeCallback[T]] = None
```

Comment:
is this intended? if so, it would deserve a comment maybe?

Reply:
It's intended, but I added a comment.